docs/custom-resources.md

```diff
@@ -15,6 +15,30 @@ Following CRDs enables users to declaratively configure all aspects of Kong:
 - [**KongCredential**](#kongcredential): These resources map to
   credentials (key-auth, basic-auth, etc) that belong to consumers.
 
+**IMPORTANT NOTE**: Kong Custom Resources are using the `kubernetes.io/ingress.class` annotation which defaults to `kong`. In
+case you have more than one Kong Ingress deployed on your cluster, make sure you label all your ressources with the appropriate
+annotation. For example:
```

Comment:
1. The ingress class annotation defaults to `nginx` and not `kong`, so let's update that.
2. Please respect the 80 column length as is the case in the rest of this file.
3. Let's move this note to the bottom of the file than here.
4. `KongIngress` custom resource doesn't support the `ingress.class` annotation as per this PR, so let's mention the resources that support the annotation explicitly. 
5. >  In case you have more than one ~Kong~ Ingress deployed
6. Typo > ressources